src/validator/DtoValidator.java

```diff
@@ -0,0 +1,77 @@
+package validator;
+
+import annotation.*;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+import java.time.Period;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class DtoValidator {
+    private static final String REGEX = "^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$";
+
+    public List<String> validateDto(Object dto) throws IllegalAccessException {
+        List<String> messages = new LinkedList<>();
+        Field[] declaredFields = dto.getClass().getDeclaredFields();
+        for (Field declaredField : declaredFields) {
+            declaredField.setAccessible(true);
+            if (declaredField.isAnnotationPresent(Length.class)) {
```

Comment:
Please change this method and split it into separate private methods, your method is too big

Reply:
Or even you can create separate validator classes for each annotation.

Reply:
Private methodnerov enem erevi, pahanji mej nshvac er 1 methodov